stock_minimum_shelf_life/models/res_config_settings.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Camptocamp
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = 'res.config.settings'
+
+    minimum_shelf_life = fields.Integer(
```

Comment:
@jgrandguillaume  In SAP is documented as 'minimum_remaining_shelf_life'. That would be better understood? Also, why document it at company level? Why not maintain it at product and product category levels?